src/pages/homepage/homepage.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import '../../styles.css';
+import './homepage.css';
+import Sidebar from '../../components/sidebar/sidebar';
```

Comment:
try to put all the components in an `index.js` file in the components folder and import the components from there. would be better for readability

Reply:
Done, thanks for mentioning this.